test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -671,7 +671,7 @@ def forward(self, x):
             match_nodes = 19
             self._test_common(mod, (v,), match_count, match_nodes, rtol=1e-2, atol=1e-2)
 
-    def _qconv2d_cpu_test_helper(self, device="cpu", int8_mixed_bf16=False):
+    def test_qconv2d_test_helper(self, device="cpu", int8_mixed_bf16=False):
```

Comment:
Could we keep this function name starting with underline? Otherwise, it will be collected and duplicated with `test_qconv2d_cpu`.

Reply:
Yes, the name should be started with an underscore, it's a mistake that I change the name starting from `test`.